### PR TITLE
Allow seconds in cron string settings

### PR DESCRIPTION
**Pull request by nezhar**
The syntax for specifying seconds in a cron job is not standardized, as traditional Unix cron does not include a field for seconds. For more information, see <https://en.wikipedia.org/wiki/Cron#Cron_expression>.

We currently use two libraries with different approaches to specifying seconds:

- `croniter`: <https://github.com/kiorky/croniter/blob/master/src/croniter/tests/test_croniter.py#L39>
  * `* * * * * */15: Every 15 seconds`
- `cron-descriptor`: <https://github.com/Salamek/cron-descriptor/blob/master/tests/TestFormats.py#L59>
  * `*/15 * * * * *: Every 15 seconds`

This pull request removes the cron description from the Admin interface and drops the `cron-descriptor` library from the requirements. Instead, the syntax of `croniter` is used, as it ensures that configurations without seconds are still compatible.

### CODE REVIEW

File: django_future_tasks/fields.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        raise ValidationError("Entry has to consist of 5 or 6 columns")
+
+    pattern = r"^(\*|\d+(-\d+)?(,\d+(-\d+)?)*)(/\d+)?$"
+    p = re.compile(pattern)
+    for i, c in enumerate(columns):
+        if not p.match(c):
+            raise ValidationError("Incorrect value {} in column {}".format(c, i + 1))
+
+
+class FuturetaskCronField(CronField):
```
Comment (galipnik) [inline django_future_tasks/fields.py:26]: For the sake of consistency I would suggest
```suggestion
class FutureTaskCronField(CronField):
```

File: django_future_tasks/migrations/0007_alter_periodicfuturetask_cron_string.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        ("django_future_tasks", "0006_periodicfuturetask_end_time"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="periodicfuturetask",
+            name="cron_string",
+            field=django_future_tasks.fields.FuturetaskCronField(
+                default="* * * * *",
+                help_text="Minute Hour Day Month Weekday",
```
Comment (galipnik) [inline django_future_tasks/migrations/0007_alter_periodicfuturetask_cron_string.py:19]: Seconds are missing here.

**Comment by galipnik** [comment]
Apart from the two comments above: LGTM.

**Review by galipnik — CHANGES_REQUESTED** [review]

File: django_future_tasks/migrations/0007_alter_periodicfuturetask_cron_string.py (hunk 2 above)
Comment (nezhar) [reply django_future_tasks/migrations/0007_alter_periodicfuturetask_cron_string.py]: Fixed

File: django_future_tasks/fields.py (hunk 1 above)
Comment (nezhar) [reply django_future_tasks/fields.py]: Fixed

**Review by galipnik — APPROVED** [review]
LGTM.

**Pull request merged**